### ISSUE TITLE
`NestedMeta` doesn't implement `Debug`

### ISSUE THREAD

**Issue by gussmith23** [d-e-s-o/test-log#22]
Just started getting this error!

```
   Compiling test-log v0.2.9
error[E0277]: `NestedMeta` doesn't implement `Debug`
  --> /Users/gus/.cargo/registry/src/github.com-1ecc6299db9ec823/test-log-0.2.9/src/lib.rs:87:58
   |
87 |     _ => panic!("unsupported attributes supplied: {:?}", args),
   |                                                          ^^^^ `NestedMeta` cannot be formatted using `{:?}` because it doesn't implement `Debug`
   |
   = help: the trait `Debug` is not implemented for `NestedMeta`
   = note: this error originates in the macro `$crate::const_format_args` (in Nightly builds, run with -Z macro-backtrace for more info)

For more information about this error, try `rustc --explain E0277`.
error: could not compile `test-log` due to previous error
```

**Comment by gussmith23** [comment]
Occurring with both
```
rustc 1.56.1 (59eed8a2a 2021-11-01)
```
and
```
rustc 1.59.0 (9d1b2106e 2022-02-23)
```

**Comment by gussmith23** [comment]
Can temporarily avoid error by pinning `test-log = "=0.2.8"`

**Comment by d-e-s-o** [comment]
Oh, hm, that's weird. I've yanked `0.2.9` for now. Thanks for reporting this issue, will look at it later.

**Comment by d-e-s-o** [comment]
I published `0.2.10` which should fix the issue. Let me know if you still see problems.
